src/main/java/BasketInformations.java

```diff
@@ -0,0 +1,98 @@
+package com.company;
+
+import java.util.HashMap;
+
+/**
+ * This class is a basket
+ */
+public class BasketInformations {
+
+	// The product of the basket
+	static HashMap<String, Integer> map = new HashMap<String, Integer>();
+
+	static String[] localProducts = new String[]{"123", "222", "44", "657"};
+
```

Comment:
I woud hide this information and change the scope to private. Also, considering we are on Java 11, you coul use the `List.of` factory method instead using an array. 

This woul give flexibility if you ant to know if a local product exist.